.env.example

```
@@ -1,57 +1,88 @@
-# VC Verification Service configs
-VC_VERIFICATION_SERVICE_URL=
-VC_DEFAULT_ISSUER_NAME=
+# Application Configuration
+NODE_ENV=development                    
+PORT=3000                              
+
+# Base64-encoded 32-byte key for AES-256-GCM encryption
+ENCRYPTION_KEY=your_base64_encoded_32_byte_key  
+
+# Database Configuration
+DB_TYPE=postgres                       
```

dotenv-linter (3.3.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)